packages/tree-widget/README.md

```diff
@@ -180,3 +180,20 @@ export class TreeWidgetControl extends WidgetControl {
   }
 }
 ```
+
+### Using the TreeWidgetUiItemsProvider
+
+The `TreeWidgetUiItemsProvider` functions like the `TreeWidgetControl` but is instead passed through the `uiProviders` prop of the `<Viewer>` component.
```

Comment:
We still need to initialize TreeWidget. So add that to the docs here, and update TreeWidget.initialize to be a lazy init, just in case it tries to get initialized more than once.